packages/vm/tests/api/EIPs/eip-3074-authcall.spec.ts

```diff
@@ -0,0 +1,680 @@
+import tape from 'tape'
+import {
+  Address,
+  bigIntToBuffer,
+  bufferToBigInt,
+  ecsign,
+  keccak256,
+  privateToAddress,
+  setLengthLeft,
+  toBuffer,
+  zeros,
+} from 'ethereumjs-util'
+import Common, { Chain, Hardfork } from '@ethereumjs/common'
+import VM from '../../../src'
+import { Transaction } from '@ethereumjs/tx'
+import { Block } from '@ethereumjs/block'
+import { ERROR } from '../../../src/exceptions'
+import { InterpreterStep } from '../../../src/evm/interpreter'
+
+const common = new Common({
+  chain: Chain.Mainnet,
+  hardfork: Hardfork.London,
+  eips: [3074],
+})
+
+// setup the accounts for this test
+const privateKey = Buffer.from(
+  'e331b6d69882b4cb4ea581d88e0b604039a3de5967688d3dcffdd2270c0fd109',
+  'hex'
+)
+
+const block = Block.fromBlockData(
+  {
+    header: {
+      baseFeePerGas: BigInt(7),
+    },
+  },
+  { common }
+)
+
+const callerPrivateKey = Buffer.from('44'.repeat(32), 'hex')
+const callerAddress = new Address(privateToAddress(callerPrivateKey))
+const PREBALANCE = BigInt(10000000)
+
+const address = new Address(privateToAddress(privateKey))
+const contractAddress = new Address(Buffer.from('ff'.repeat(20), 'hex'))
+const contractStorageAddress = new Address(Buffer.from('ee'.repeat(20), 'hex'))
+
+// Bytecode to exit call frame and return the topmost stack item
+const RETURNTOP = Buffer.from('60005260206000F3', 'hex')
+// Bytecode to store CALLER in slot 0 and GAS in slot 1 and the first 32 bytes of the input in slot 2
+// Returns the entire input as output
+const STORECALLER = Buffer.from('5A60015533600055600035600255366000600037366000F3', 'hex')
+
+/**
+ * This signs a message to be used for AUTH opcodes
+ * @param commitUnpadded - The commit which we want to sign
+ * @param address - The contract address we are using AUTH on
+ * @param privateKey - The private key of the account to sign
+ * @returns - Returns the signed message
+ */
+function signMessage(commitUnpadded: Buffer, address: Address, privateKey: Buffer) {
+  const commit = setLengthLeft(commitUnpadded, 32)
+  const paddedInvokerAddress = setLengthLeft(address.buf, 32)
+  const chainId = setLengthLeft(bigIntToBuffer(common.chainId()), 32)
+  const message = Buffer.concat([Buffer.from('03', 'hex'), chainId, paddedInvokerAddress, commit])
+  const msgHash = keccak256(message)
+  return ecsign(msgHash, privateKey, 0)
+}
+
+/**
+ * This method returns the bytecode in order to set AUTH
+ * @param commitUnpadded - The commit
+ * @param signature - The signature as obtained by `signMessage`
+ * @returns
```

Comment:
empty `@returns` here (and L110), you can just remove and typedoc knows it returns a Buffer, unless you want to add more description (same with the params). i do know this is in a test file though so just a nit :)